app/portal/horizon/requirements.txt

```diff
@@ -10,7 +10,7 @@
 pbr!=2.1.0,>=2.0.0 # Apache-2.0
 # Horizon Core Requirements
 Babel!=2.4.0,>=2.3.4 # BSD
-django>=2.2.13, <=2.2.13
+django>=2.2.16, <=2.2.16
```

Comment:
Remediation suggestion was to upgrade after django>=2.2.18, shall we check the version after it? the 2.2.18 fixed a moderate security issue.

Reply:
done.

Reply:
> Remediation suggestion was to upgrade after django>=2.2.18, shall we check the version after it? the 2.2.18 fixed a moderate security issue.

If everything goes the same as 2.2.16, we'd better change it to 2.2.16.